lint issues

i don't know what is going on, but eslint has started complaining 

```
/Users/nickm/git/whisk/nickm/kui-master/kui/packages/app/src/core/symbol-table.ts
  31:3  error  Missing accessibility modifier on method definition read   @typescript-eslint/explicit-member-accessibility
  35:3  error  Missing accessibility modifier on method definition write  @typescript-eslint/explicit-member-accessibility
```

--- a/.eslintignore
+++ b/.eslintignore
@@ -3,6 +3,7 @@ packages/kui-builder
 clients/**/*/dist/
 plugins/**/*/dist/
 plugins/*/lib/**/*.js
+plugins/*/i18n/**/*.js
 plugins/*/*.js
 packages/app/dist/
 **/*/node_modules/

--- a/packages/app/src/core/command-tree.ts
+++ b/packages/app/src/core/command-tree.ts
@@ -23,7 +23,6 @@ import {
   CommandTree,
   CommandTreeResolution,
   Disambiguator,
-  EvaluatorArgs,
   ExecType,
   CatchAllOffer,
   CatchAllHandler,

--- a/packages/app/src/core/symbol-table.ts
+++ b/packages/app/src/core/symbol-table.ts
@@ -28,11 +28,11 @@ class SymbolTable {
     return JSON.parse(sessionStore().getItem(this.symbolTableSessionStorageKey)) || {}
   }
 
-  static read(tab: Tab): Record<string, string> {
+  public static read(tab: Tab): Record<string, string> {
     return this.getSymbolTable()[getTabId(tab)] || {}
   }
 
-  static write(tab: Tab, curDic: Record<string, string>) {
+  public static write(tab: Tab, curDic: Record<string, string>) {
     const storage = this.getSymbolTable()
     storage[getTabId(tab)] = curDic
     sessionStore().setItem(this.symbolTableSessionStorageKey, JSON.stringify(storage))

--- a/plugins/plugin-bash-like/src/lib/cmds/export.ts
+++ b/plugins/plugin-bash-like/src/lib/cmds/export.ts
@@ -15,8 +15,6 @@
  */
 
 import { CommandRegistrar, EvaluatorArgs } from '@kui-shell/core/models/command'
-import sessionStore from '@kui-shell/core/models/sessionStore'
-import { getTabId, getCurrentTab } from '@kui-shell/core/webapp/cli'
 import SymbolTable from '@kui-shell/core/core/symbol-table'
 
 /**

--- a/plugins/plugin-core-support/src/lib/new-tab.ts
+++ b/plugins/plugin-core-support/src/lib/new-tab.ts
@@ -357,8 +357,6 @@ const newTab = async (basedOnEvent = false): Promise<boolean> => {
   const currentVisibleTab = getCurrentTab()
   ;(currentVisibleTab['state'] as TabState).capture()
 
-  const nTabs = document.querySelectorAll('.main > .tab-container > tab').length
-
   const newTab = currentVisibleTab.cloneNode(true) as HTMLElement
   newTab.className = 'visible'
 
@@ -414,7 +412,6 @@ const newTab = async (basedOnEvent = false): Promise<boolean> => {
 const oneTimeInit = (): void => {
   const initialTab = getTabFromIndex(1)
   const initialTabButton = getCurrentTabButton()
-  const initialTabId = initialTabButton.getAttribute('data-tab-id')
 
   if (document.body.classList.contains('subwindow')) {
     element(tabButtonSelector).onclick = () => window.open(window.location.href, '_blank')
@@ -456,8 +453,7 @@ const newTabAsync = ({ execOptions }: EvaluatorArgs) => {
 const registerCommandHandlers = (commandTree: CommandRegistrar) => {
   commandTree.listen(
     '/tab/switch',
-    ({ argvNoOptions, parsedOptions }) =>
-      switchTab(getTabId(getTabFromIndex(parseInt(argvNoOptions[argvNoOptions.length - 1], 10)))),
+    ({ argvNoOptions }) => switchTab(getTabId(getTabFromIndex(parseInt(argvNoOptions[argvNoOptions.length - 1], 10)))),
     { usage, needsUI: true, noAuthOk: true }
   )
   commandTree.listen('/tab/new', newTabAsync, {

